tests/pool_test.go

```diff
@@ -157,7 +157,7 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
 		return createVirtualMachinePool(newPoolFromVMI(libvmifact.NewCirros()))
 	}
 
-	DescribeTable("pool should scale", func(startScale int, stopScale int) {
+	DescribeTable("pool should scale", decorators.Conformance, func(startScale int, stopScale int) {
```

Comment:
Unfortunately this is not GA API, please drop this commit (the first one is still welcome)